CONTRIBUTING.md

```diff
@@ -0,0 +1,272 @@
+# Contributing to Blockstack.org
+
+Looking to contribute something to Blockstack? **Here's how you can help.**
+
+Please take a moment to review this document in order to make the contribution process easy and effective for everyone in the community.
+
+Following these guidelines helps to communicate that you respect the time of everyone involved in the Blockstack open source community. In return, the community will address your issue or assess patches and features as quickly as possible.
+
+## Table of contents
+
+- [Finding opportunities to contribute](#finding-opportunities-to-contribute)
+- [Using the issue tracker](#using-the-issue-tracker)
+- [Bug reports](#bug-reports)
+- [Feature requests](#feature-requests)
+- [Pull requests](#pull-requests)
+- [Code guidelines](#code-guidelines)
+- [Maintainers](#maintainers)
+- [License](#license)
+
+## Finding opportunities to contribute
+
+There are two common ways to find opportunities to contribute:
+
+- [Project Boards](https://github.com/blockstack/blockstack.org/projects)
+- [Issue Tracker](https://github.com/blockstack/blockstack.org/issues)
+
+The project boards is a great place to start. Look for the version that has the label `current`
+and look for a ticket in the `To Do` column. Click on the ticket your interested in
+and this will link you to the relative issue in the tracker.
+
+If the ticket your interested in is being worked, feel free to provide feedback.
+
+Otherwise, leave a comment addressing one of the [repo maintainers](#maintainers) to get
+the conversation started.
+
+Whether you are a veteran programmer, beginner programmer, or don't program at all, we
+are positive you will find a place to helping us build a decentralized internet!
+
+[Just Ask!](http://chat.blockstack.org/)
+
+[^ Back To The Top](#contributing-to-blockstackorg)
+
+## Using the issue tracker
+
+The [issue tracker](https://github.com/blockstack/blockstack.org/issues) is the preferred channel for [bug reports](#bug-reports), [features requests](#feature-requests) and [submitting pull requests](#pull-requests), but please respect the following
+restrictions:
+
+* Please **do not** use the issue tracker for personal support requests.  Please use the [Forum](https://forum.blockstack.org) or [Slack](http://chat.blockstack.org) as they are better places to get help.
+
+* Please **do not** derail or troll issues. Keep the discussion on topic and respect the opinions of others.
+
+* Please **do not** post comments consisting solely of "+1" or ":thumbsup:". Use [GitHub's "reactions" feature](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments) instead. We reserve the right to delete comments which violate this rule.
```

Comment:
we could leave it in so contributors can at least work try and follow it, I can remove the right to delete comments that do this and just not enforce this. But again I think it is nice to have because it is a cleaner way to show support rather than scrolling through a page of +1's